flutter/README.md

```diff
@@ -1 +1,26 @@
-TODO(matuella): Add this doc
\ No newline at end of file
+# Flutter
+
+Source files for Memo's Flutter application.
+
+> At this moment, Memo only output builds for Android and iOS, although - **eventually** - there may be a support for 
+> desktop (Windows, Linux and macOS) and web, all using Flutter!
+
+## Setup
+
+1. If you have no idea how to install Flutter and run it locally, check this
+[_Get started_](https://flutter.dev/docs/get-started/install).
+2. If you have Flutter setup locally, install pubspec dependencies by running `flutter pub get` in `flutter/` root
+folder.
+3. Setup Firebase dependencies. More information on this in [ARCHITECTURE#firebase](flutter/ARCHITECTURE.md#firebase).
+
+
+... and you're good to go - just run in any physical device, or a simulator/emulator.
```

Comment:
I don't know if physical device will work for iOS without provision profiles setup

Reply:
Why not? It can simply ignore the provisions and change to its own developer account

Reply:
But if the user needs an extra step to launch, shouldn't it be noted here?

Reply:
I think this goes a little bit off of this explanation, as it's related to the setup of this project in particular, but I do get your point. I'm open to change it, although my opinion is to not add it. What's your suggestion on this one?

Reply:
I would add an extra step:

`4. [Optional] To run in physical iOS devices, it's necessary to setup the provision profiles accordingly. You may change them by opening the XCode project and updating the Development Team in the target Signing & Capabilities to your personal developer account.`

What do you think?

Reply:
I think it's pretty verbose, but I will try to make them more succinct while keeping the idea